### PR TITLE
[docs] Introduce video introduction

### PR DESCRIPTION
**Pull request by jugglinmike**
Compared to the existing text, a visual introduction may be more
approachable for some potential contributors. The documentation
currently recommends a recording of a lecture, but that video is too
long and technical to reach those whose commitment to the project is
uncertain.

Introduce a new video designed for first-time contributors. Include a
transcript so that visitors who are not able to experience the video may
still benefit from its content. Relocate the link to the lecture to the
appendix.

---

Like gh-17811 and gh-17894, this was inspired by conversations with @fantasai, @frivoal, @lmccart, @ericholscher, @jihyerish, and @chrisdavidmills and [the experience of the Django project maintainers](https://jacobian.org/2009/nov/10/what-to-write/).

This work-in-progress pull request currently only contains a transcript; we haven't made the video yet. My hope is that this will make it easier to design the content collaboratively. Once those involved are satisfied, I'll follow up by producing the actual recording.

Does this hit all the most important points for a first-time contributor? Is there anything we should add? Anything we should remove?

### CODE REVIEW

**Review by sideshowbarker — APPROVED** [review]

File: docs/appendix/intro-video-transcript.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+> The latest version of the tests are publicly hosted in executable form on the
+> web at web-platform-tests.live.
+
+The browser begins scrolling through the enormous list of directories.
+
+> There, were can navigate among all the tests for all the different web
+> technologies.
+>
+> Let's take a look at a typical test.
```
Comment (lukebjerring) [inline docs/appendix/intro-video-transcript.md]: Might be good to include typing in a search (e.g. `fetch`)

File: docs/appendix/intro-video-transcript.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+source of the failing test.
+
+> It looks like the failing subtest is for what happens when a `Headers`
+> instance has a custom JavaScript iterator method. That's a strange edge case,
+> but it's important for browsers to agree on every detail!
+
+`https://web-platform-tests.org` is entered into the location bar, and the
+browser loads the page.
+
+> Other types of tests include "reftests" (for verifying the visual rendering
```
Comment (lukebjerring) [inline docs/appendix/intro-video-transcript.md]: Maybe move to a reftest on wpt.fyi for visual effect here?

**Review by lukebjerring — COMMENTED** [review]
Feels concise enough to keep attention, and informative enough to be educational!

File: docs/appendix/intro-video-transcript.md (hunk 2 above)
Comment (jugglinmike) [reply docs/appendix/intro-video-transcript.md]: Good idea! I'm suggesting more "view source" stuff to demonstrate the relation between test and reference. I'll add some more explanation according to what feels natural in the time it takes to perform this action.

**Comment by jugglinmike** [comment]
Thanks for the review, @lukebjerring! I've selected some moderately interesting examples for reference tests and manual tests, but there are probably better ones out there. Do you have any favorites?

File: docs/appendix/intro-video-transcript.md [hunk 3]
```diff
@@ -118,12 +118,24 @@ source of the failing test.
 > instance has a custom JavaScript iterator method. That's a strange edge case,
 > but it's important for browsers to agree on every detail!
 
+`http://web-platform-tests.live/css/css-transforms/transform-transformed-tr-contains-fixed-position.html`
```
Comment (lukebjerring) [inline docs/appendix/intro-video-transcript.md]: https://wpt.fyi/results/css/css-transforms/transform-transformed-tr-contains-fixed-position.html highlights the "reference vs actual" concept a little more here.

**Review by lukebjerring — COMMENTED** [review]
No favourites - but I'll forward on the question in case someone does have one or two.

**Review by gsnedders — COMMENTED** [review]
this basically looks fine to me

File: docs/appendix/intro-video-transcript.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+  frameborder="0"
+  allow="autoplay; encrypted-media"
+  allowfullscreen></iframe>
+
+**Still image of the WPT logo**
+
+> Hello, and welcome to the Web Platform Tests!
+>
+> The goal of this project is to ensure that all web browsers present websites
+> in exactly the way the websites' authors intended.
```
[the remark this answers is not in the document]
Comment (jugglinmike) [reply docs/appendix/intro-video-transcript.md]: You got it

**Comment by jugglinmike** [comment]
@sideshowbarker @lukebjerring @gsnedders The video's ready! As expected, the actual timing of the action informed the spoken word a bit. The video includes a more substantial description of reftests and manual tests than originally proposed.

What do you think?

**Comment by jugglinmike** [comment]
Here's a direct link: https://www.youtube.com/watch?v=SMdEIRcolSY

**Comment by jugglinmike** [comment]
Some folks let me know that the volume of the narration is too low from 2:44 until 4:08. Creating a new version takes a little time, so I'm going to hold off on fixing that while waiting for more feedback.

@lukebjerring Are you purchasing wpt.live to replace web-platform-tests.live? If so, I should re-record the parts that reference the latter.

**Comment by lukebjerring** [comment]
Yes, @foolip already did I think? and wpt.reviews is also the plan, just jumping through bureaucracy hoops to make that happen!

**Comment by foolip** [comment]
Yep, we've had wpt.live and not-wpt.live since February.

**Comment by jugglinmike** [comment]
@foolip @gsnedders @sideshowbarker I've re-recorded the audio and updated the reference to wpt.live. This should be good to go!

**Review by foolip — APPROVED** [review]
I just watched this for the first time, I loved it!

**Comment by jugglinmike** [comment]
That's great, @foolip! Do you think we should seek out approval from anyone in particular?

**Comment by foolip** [comment]
@jugglinmike not really, plenty of people were poked and two have explicitly approved, so I'll just go ahead and merge this.

When you've put the project files and highest quality render somewhere, can you link to them in a comment or something, in case we don't want to use YouTube at some point in the future?

**Comment by jugglinmike** [comment]
You got it

Video: https://archive.org/details/wpt-intro-video-2019

Assets: https://archive.org/details/wpt-intro-video-2019-assets

**Comment by foolip** [comment]
Excellent, thank you!
